src/SGFTree.cpp

```diff
@@ -477,6 +477,10 @@ std::string SGFTree::state_to_string(GameState& pstate, int compcolor) {
         }
     }
 
+    auto tc = state->get_timecontrol();
+    header.append("\nC[LeelaZ options: " + cfg_options_str + "\n");
+    header.append("LeelaZ time control: " + tc.to_string() + "]");
```

Comment:
SGF has TM and OT properties to describe the timecontrol. We should be good citizens and use those.

Reply:
Ah right, I'll fix this and the PROGRAM_NAME tonight.

Reply:
Fixed.